### PR TITLE
Jenkins changes to run process tests

### PR DESCRIPTION
**Pull request by susannasiebert**

### CODE REVIEW

File: bin/process-test-multi [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+
+set_genome_software_result_test_name();
+
+# Done with Per Process Customizations
+print "\n";
+
+my $process = get_initial_process();
+wait_for_process($process);
+
+$process = UR::Context->current->reload('Genome::Process', id => $process->id);
```
Comment (nnutter) [inline bin/process-test-multi]: I think you can do `$process = UR::Context->current->reload($process)` or maybe even just `UR::Context->current->reload($process)`.

**Comment by nnutter** [comment]
There seems to be a lot of code from `model-test-multi` being duplicated here.

File: bin/process-test-multi (hunk 1 above)
Comment (nnutter) [inline bin/process-test-multi]: Just noticed you're even using that style later.

**Comment by susannasiebert** [comment]
Test failure is because this PR depends on https://github.com/genome/genome/pull/589

**Comment by apregier** [comment]
One point of discussion - I originally thought that process tests and model tests would run in the same workspace, but now I am thinking it would be easier to keep them separate.  Once we turn on the test-db, it seems to me that it would be easier to maintain separate templates for process tests and model tests.  @nnutter do you have an opinion about that?

**Comment by nnutter** [comment]
@apregier I think separate templates is reasonable.  To facilitate that we need to decide on a way to configure the default template.  I think it needs to be strongly coupled to the source code, e.g. in Git, but I don't like the way it is right now.

**Comment by susannasiebert** [comment]
@nnutter Allison made some changes to deduplicate the code.

**Comment by susannasiebert** [comment]
@nnutter: I just merged in a PR by Allison that addresses your comments.

**Comment by nnutter** [comment]
:+1: I think the test failures may be due to GitHub getting DDoS'd.

**Comment by susannasiebert** [comment]
Looks like it but it would probably fail either way because genome/genome#589 hasn't been merged yet.

**Comment by nnutter** [comment]
One of the test failures looks like it's not just due to genome/genome#589 not being merged,

```
14:53:36 t/..//bin/make_snapshot syntax OK
14:53:37 "my" variable $build masks earlier declaration in same scope at t/..//bin/model-test-multi line 24.
14:53:37 "my" variable $build masks earlier declaration in same scope at t/..//bin/model-test-multi line 26.
14:53:37 "my" variable $build masks earlier declaration in same statement at t/..//bin/model-test-multi line 26.
14:53:37 "my" variable $build masks earlier declaration in same scope at t/..//bin/model-test-multi line 28.
14:53:37 "my" variable $build masks earlier declaration in same scope at t/..//bin/model-test-multi line 30.
14:53:37 syntax error at t/..//bin/model-test-multi line 19, near ") {"
14:53:37 t/..//bin/model-test-multi had compilation errors.
14:53:37 
14:53:37 #   Failed test 't/..//bin/model-test-multi compiled'
14:53:37 #   at t/compile.t line 17.
```

**Comment by susannasiebert** [comment]
@nnutter Fixed. Thanks for catching that.

**Pull request merged**